### PR TITLE
Adding hindi translation

### PR DESCRIPTION
**Pull request by gryptoo**
Pull Request type
Documentation content changes

What is the current behavior?
Resolves: https://github.com/keep-starknet-strange/madara/issues/926

What is the new behavior?
madara beast article.md is translated to hindi.

Does this introduce a breaking change?
No

### CODE REVIEW

**Comment by tdelabro** [comment]
@apoorvsadana Is it something you can review?

**Comment by apoorvsadana** [comment]
Hey @tdelabro, sure, I will try to review this in some time.

File: docs/content/articles/hn/madara-beast-article.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Substrate framework स्वाभाव से modular nature का है, जो developers को आसानी है appchains को customize करने की अनुमति देता है. कोई assumptions लागू नहीं होते, आप अपने हिसाब से consensus protocol, hash function, signature scheme, storage layout को चुन सकते हैं – जो भी आपको चाहिए, वो भी साथ में Cairo का इस्तेमाल करके Proofs generate करने के लिए. Developers बिना पाबंदी के कुछ भी कर सकते हैं, वो भी provable रहते हुए, और underlying chain की security को inherit करते हुए, चाहे वो Starknet हो या Ethereum. 
+
+शुरुआत में, Madara Starknet से काफी सामान लगेगा, ताकि Starknet ecosystem के अंदर smart contracts की composability मुमकिन हो. आगे चलके, Starknet का [Herodotus](https://www.herodotus.dev/) से integration होगा तो [Storage Proofs](https://book.starknet.io/chapter_8/storage_proofs.html) का इस्तेमाल होगा interoperability के लिए. Storage Proofs की मदद से Madara appchains दुसरे chains की state और liquidity को भी consider कर पाएंगे.
```
Comment (apoorvsadana) [inline docs/content/articles/hn/madara-beast-article.md:69]: सामान doesn't seem to be the right word here, effort maybe?

**Comment by apoorvsadana** [comment]
@gryptoo I have added some comments. Along with those comments, I noticed we are using exact English words in a lot of places. That's fine if we are using proper nouns like Starkware and Madara or techincal terms like L1s or validity proofs. But for general English words like `consider`, `generate` etc. we should ideally use the Hindi words. Otherwise, lgtm

**Comment by d-roak** [comment]
@gryptoo can you take a look at the comments, please? Otherwise, @apoorvsadana can you take over this pr?
If there are no updates within the next week, I think we should close this cc @tdelabro

**Comment by gryptoo** [comment]
my bad, missed the previous comments. I'd used general english terms for words that aren't really used in colloquial hindi and would make the article confusing imo

**Comment by apoorvsadana** [comment]
@gryptoo, can you refer to which words you were talking about. I feel `consider` and `generate` have easy to understand hindi terms. Also, can you reply to the comments above as well regarding the typos/suggestions

**Pull request closed without merging**